docs/sources/write/front-matter/index.md

```diff
@@ -62,7 +62,7 @@ For example, `date: "2023-04-24T00:00:00Z"` is 12:00 AM, Apr 24 Coordinated Univ
 The `date` front matter also impacts menu ordering.
 Pages with more recent dates are lower in the menu.
 
-### description (required)
+### `description` (required)
 
 Use to provide the short description of the topic to search engines, including the search engine used in the Grafana documentation site. The description is also displayed on social media, such as Twitter, to provide a clue to users about the page contents.
```

Comment:
change name to X?

Reply:
Lol! I'm not sure Twitter has lost meaning yet. At some point this change might be needed but I don't think it's necessary yet.